.github/check-js.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+#
+# This file is part of the phpBB Forum Software package.
+#
+# @copyright (c) phpBB Limited <https://www.phpbb.com>
+# @license GNU General Public License, version 2 (GPL-2.0)
+#
+# For full copyright and license information, please see
+# the docs/CREDITS.txt file.
+#
+set -e
+set +x
+
+sudo npm install -g > /dev/null
+npm ci > /dev/null
+set -x
+# node_modules/eslint/bin/eslint.js "phpBB/adm/style/*.js" # Currently disabled as no file is linted
+node_modules/eslint/bin/eslint.js "phpBB/assets/javascript/*.js"
+# node_modules/eslint/bin/eslint.js "phpBB/style/all/js/*.js" # Currently disabled as no file is linted
+# node_modules/eslint/bin/eslint.js "phpBB/style/prosilver/template/*.js" # Currently disabled as no file is linted
+node_modules/eslint/bin/eslint.js "gulpfile.js"
```

Comment:
why are these commented out? This will prevent us from listing any new files...ie mentions.js which should be linted

Reply:
I've commented these out cause eslint will otherwise display an error message along the lines of "Oops, couldn't run any tests cause everything is currently being filtered out".

Reply:
Because you globally ignored all js files mentions is being ignored. But there is probably a flag to disable it

Reply:
Line 18 should actually lint mentions.